src/helpers/figures/charts/chart_ui_common.ts

```diff
@@ -231,9 +251,12 @@ export function getChartDatasetValues(getters: Getters, dataSets: DataSet[]): Da
           ? truncateLabel(cell.formattedValue)
           : (label = `${ChartTerms.Series} ${parseInt(dsIndex) + 1}`);
     } else {
-      label = label = `${ChartTerms.Series} ${parseInt(dsIndex) + 1}`;
+      label = `${ChartTerms.Series} ${parseInt(dsIndex) + 1}`;
     }
     let data = ds.dataRange ? getData(getters, ds) : [];
+    if (data.every((e) => typeof e === "string" && !e.startsWith("#"))) {
+      data = countStringOccurrences(data).values;
```

Comment:
aggregating the values should be done in `aggregateDataForLabels`, shouldn't it?